common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -32,6 +32,15 @@
 from embargo.test_utils import restrict_course
 
 
+class UrlMixin(object):
+    """
+    Mixins for URL processing
+    """
+    def get_absolute_url(self, path):
+        """ Generate an absolute URL for a resource on the test server. """
+        return u'http://testserver/{}'.format(path.lstrip('/'))
+
+
```

Comment:
@jimabramson This code was actually already implemented in this test class. I didn't write it, I just copied it into a mixin under the recommendation of @clintonb. So are you suggesting that the HttpRequest method should replace this mixin altogether?